specs/proposal/542678_improve_bundle_image_provided_shape/improve_bundle_image_provided_shape.textile

```diff
@@ -50,8 +50,10 @@ h3. Documentation Changes
 
 h2. Tests and Non-regression strategy
 
-We will need to test the color, border color and border size with two new sample:
+Tests on the older version of the extension point need to be still working after this enhancement.
+
+For the new extension point, we will need to test the color, border color and border size with two new sample:
```

Comment:
> with two new sample

with 3 new sample**s**

Reply:
I just have two new samples: 
- one with multi-valued attributes different than from donut.svg
- one with single-valued attributes

Reply:
After some toughts, I will also had the donut.svg sample. It will not make a difference with the other multi-valued sample...but it will draw a connection between the old extension point and the new one.